v2/azure-arm.yaml

```diff
@@ -31,6 +31,7 @@ supportedResourcesReport:
   # resourcePath is a template used for generating a file path for checking whether docs for a resource have been generated
   # specified relative to the directory of outputPath, and supports the same placeholders
   resourcePathTemplate: "{group}/{version}.md"
+  currentRelease: v2.1.0
```

Comment:
Also, can consider adding a yq cmd like below in `generate-types` task target: 

`yq eval -i  ".currentRelease=\"{LATEST_VERSION_TAG}\"" ./v2/azure-arm.yaml`

Reply:
Thanks for the command, it's pretty close to what I want to put into the CRD Docs workflow.

But wouldn't the path need to be `.supportedResourcesReport.currentRelease` ?